app/models/onboarding_alert.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: alerts
+#
+#  id             :integer          not null, primary key
+#  course_id      :integer
+#  user_id        :integer
+#  article_id     :integer
+#  revision_id    :integer
+#  type           :string(255)
+#  email_sent_at  :datetime
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  message        :text(65535)
+#  target_user_id :integer
+#  subject_id     :integer
+#  resolved       :boolean          default(FALSE)
+#  details        :text(65535)
+#
+
+# Alert for from where the instructor learnt about Wiki Edu.
+class OnboardingAlert < Alert
+  def main_subject
+    'WikiEdu: Onboarding Alert'
```

Comment:
I suggest something more like "Onboarding Alert: [username] / [real name]"

This is the subject line of the email, so seeing the username and real name of the user will make these more useful.